cmd/buildah/manifest.go

```diff
@@ -237,6 +251,36 @@ func init() {
 	manifestCommand.AddCommand(manifestRmCommand)
 }
 
+func manifestExistsCmd(c *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errors.New("At least a name must be specified for the list")
+	}
+	name := args[0]
+
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	systemContext, err := parse.SystemContextFromOptions(c)
+	if err != nil {
+		return fmt.Errorf("error building system context: %w", err)
+	}
+	runtime, err := libimage.RuntimeFromStore(store, &libimage.RuntimeOptions{SystemContext: systemContext})
+	if err != nil {
+		return err
+	}
+
+	_, err = runtime.LookupManifestList(name)
+	if err != nil {
+		if errors.Is(err, storage.ErrImageUnknown) {
+			os.Exit(1)
```

Comment:
This bypasses the store shutdown logic, which has to be run if we initialized a store, if only to undo the private bind mount of the storage root over itself.  I'm surprised this doesn't break in CI, which tends to have trouble removing that directory when that happens.

Reply:
Added a call to `shutdownStore` just before we exit manually, will this address the mentioned issue.

Reply:
I'd slightly prefer a way to have the final cleanup logic in `main()` handle this, but this should do.  I'm still wary that CI didn't catch this - the tests with the overlay driver would have done so in the past.